sdk/spring/spring-cloud-azure-stream-binder-eventhubs/src/main/java/com/azure/spring/cloud/stream/binder/eventhubs/EventHubsMessageChannelBinder.java

```diff
@@ -258,4 +269,12 @@ public void setCheckpointStore(CheckpointStore checkpointStore) {
     public InstrumentationManager getInstrumentationManager() {
         return instrumentationManager;
     }
+
+    /**
+     * Set the client factory customizers.
+     * @param clientFactoryCustomizers The client factory customizers.
+     */
+    public void setClientFactoryCustomizers(List<ClientFactoryCustomizer> clientFactoryCustomizers) {
+        this.clientFactoryCustomizers = clientFactoryCustomizers;
```

Comment:
what if the passed in argument is null? may add?

Reply:
The binder will be created via the binder auto-configuration, so the list won't be null.